### ISSUE TITLE
english

### ISSUE THREAD

**Issue by bluethefoxyt** [tor4kichi/Uncord#47]
Will you add any English options

**Comment by tor4kichi** [comment]
yes. i will english translation support with next update(v0.2). 
but i trying other project, so this addition are far of day. 
thanks your intaraction.

英語のUI表示は次のアップデートで対応を予定しています。
ただ、他のプロジェクトに取り組んでいるため、実装予定は少し後になりそうです。
Uncordに興味を持ってくれてありがとう。

**Comment by bluethefoxyt** [comment]
@tor4kichi also there is no way of sending messages you need to add a send button for sending on windows phone

**Comment by tor4kichi** [comment]
fixed on v0.1.5, both of english support and sending button in mobile. thanks report!

now shipping package release. 

thanks waiting update, and sorry too late update.
